client/src/Auth/Register/register.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+// https://www.flaticon.com/free-icon/circular-avatar_108331
+import avatar from './avatar.png';
+import './register.css';
+
+// Checks whether state values meets valid criteria
+function validateInputs ({ username, password, confirmPassword }) {
+    
+    return {
+        username: username.length < 4,
```

Comment:
Don't use magic numbers in the code. That's how you end up with discrepancies (e.g. password min length is 5 in the code, should be 7 according to error message). Define const variables at the top of the file, e.g. `const MIN-LENGTH-USERNAME = 4;`, and use them in the code like this: `username: username.length < MIN-LENGTH-USERNAME`.